AviUtl: lwi file is not created and the video cannot be read

If the file is in a format that does not require indexing, such as an mp4 file, it can be read.
There seems to be a issue after this commit (997669f).

EDIT: This issue only occurs when used as an AviUtl plugin.

I tried replacing it, but I got a build error.

<details><summary>log</summary>
<p>

```
make -j 4  
CFLAGS/LDFLAGS checking...  
checking for liblsmash...  
checking for libavformat...  
checking for libavcodec...  
checking for libswscale...  
checking for libswresample...  
~/L-SMASH-Works-HomeOfAviSynthPlusEvolution/AviUtl ~/L-SMASH-Works-HomeOfAviSynthPlusEvolution/AviUtl  
~/L-SMASH-Works-HomeOfAviSynthPlusEvolution/AviUtl  

settings...  
CC          = gcc  
LD          = gcc  
RC          = windres  
STRIP       = strip  
CFLAGS      = -Os -ffast-math -Wall -std=c99 -pedantic -D__USE_MINGW_ANSI_STDIO=1 -I. -I. -I../include  -fexcess-precision=fast -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/include -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32include  
LDFLAGS     = -shared -Wl,--dll,--add-stdcall-alias -L.  
LIBS        = -lcomctl32 -lwinmm -lgdi32 -LC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/lib -llsmash -LC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32lib -lavformat -lbz2 -lavcodec -lvpx -liconv -llzma -lz -ldav1d -lswscale -lswresample -lavutil -lm -lmfx -lstdc++ -lole32 -luuid -luser32 -lbcrypt  -static -static-libgcc -static-libstdc++  

configure finished.  
type 'make'        : compile all plugins  
type 'make input'  : compile lwinput.aui  
type 'make muxer'  : compile lwmuxer.auf  
type 'make dumper' : compile lwdumper.auf  
type 'make color'  : compile lwcolor.auc  
rm -f *.auf *.aui *.auc *.o *.res .depend  
windres -J rc -O coff lwinput.rc -o lwinput.res  
windres -J rc -O coff lwmuxer.rc -o lwmuxer.res  
gcc -Os -ffast-math -Wall -std=c99 -pedantic -D__USE_MINGW_ANSI_STDIO=1 -I. -I. -I../include  -fexcess-precision=fast -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/include -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32include  -c lwinput.c -o lwinput.o  
gcc -Os -ffast-math -Wall -std=c99 -pedantic -D__USE_MINGW_ANSI_STDIO=1 -I. -I. -I../include  -fexcess-precision=fast -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/include -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32include  -c libavsmash_input.c -o libavsmash_input.o  
gcc -Os -ffast-math -Wall -std=c99 -pedantic -D__USE_MINGW_ANSI_STDIO=1 -I. -I. -I../include  -fexcess-precision=fast -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/include -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32include  -c lwlibav_input.c -o lwlibav_input.o  
gcc -Os -ffast-math -Wall -std=c99 -pedantic -D__USE_MINGW_ANSI_STDIO=1 -I. -I. -I../include  -fexcess-precision=fast -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32/include -IC:/Users/ekusu/Downloads/lsw-build-bat-master/msys64/mingw32include  -c avs_input.c -o avs_input.o  
In file included from lwlibav_input.c:42:  
../common/lwindex.h:35:21: error: expected ':', ',', ';', '}' or '__attribute__' before '=' token  
   35 |     char *cache_dir = NULL;  
      |                     ^  
In file included from avs_input.c:36:  
../include/avisynth_c.h:123:21: warning: enumerator value for 'AVS_CS_PLANAR' is not an integer constant expression [-Wpedantic]  
  123 |     AVS_CS_PLANAR = 1 << 31,  
      |                     ^  
lwlibav_input.c: In function 'open_file':  
lwlibav_input.c:203:16: error: 'lwlibav_option_t' has no member named 'threads'  
  203 |     lwlibav_opt.threads           = opt->threads;  
      |                ^  
lwlibav_input.c:204:16: error: 'lwlibav_option_t' has no member named 'av_sync'  
  204 |     lwlibav_opt.av_sync           = opt->av_sync;  
      |                ^  
lwlibav_input.c:205:16: error: 'lwlibav_option_t' has no member named 'no_create_index'  
  205 |     lwlibav_opt.no_create_index   = opt->no_create_index;  
      |                ^  
lwlibav_input.c:206:16: error: 'lwlibav_option_t' has no member named 'index_file_path'  
  206 |     lwlibav_opt.index_file_path   = NULL;  
      |                ^  
lwlibav_input.c:207:16: error: 'lwlibav_option_t' has no member named 'force_video'  
  207 |     lwlibav_opt.force_video       = opt->force_video;  
      |                ^  
lwlibav_input.c:208:16: error: 'lwlibav_option_t' has no member named 'force_video_index'  
  208 |     lwlibav_opt.force_video_index = opt->force_video_index;  
      |                ^  
lwlibav_input.c:209:16: error: 'lwlibav_option_t' has no member named 'force_audio'  
  209 |     lwlibav_opt.force_audio       = opt->force_audio;  
      |                ^  
lwlibav_input.c:210:16: error: 'lwlibav_option_t' has no member named 'force_audio_index'  
  210 |     lwlibav_opt.force_audio_index = opt->force_audio_index;  
      |                ^  
lwlibav_input.c:211:16: error: 'lwlibav_option_t' has no member named 'apply_repeat_flag'  
  211 |     lwlibav_opt.apply_repeat_flag = opt->video_opt.apply_repeat_flag;  
      |                ^  
lwlibav_input.c:212:16: error: 'lwlibav_option_t' has no member named 'field_dominance'  
  212 |     lwlibav_opt.field_dominance   = opt->video_opt.field_dominance;  
      |                ^  
lwlibav_input.c:213:16: error: 'lwlibav_option_t' has no member named 'vfr2cfr'  
  213 |     lwlibav_opt.vfr2cfr.active    = opt->video_opt.vfr2cfr.active;  
      |                ^  
lwlibav_input.c:214:16: error: 'lwlibav_option_t' has no member named 'vfr2cfr'  
  214 |     lwlibav_opt.vfr2cfr.fps_num   = opt->video_opt.vfr2cfr.framerate_num;  
      |                ^  
lwlibav_input.c:215:16: error: 'lwlibav_option_t' has no member named 'vfr2cfr'  
  215 |     lwlibav_opt.vfr2cfr.fps_den   = opt->video_opt.vfr2cfr.framerate_den;  
      |                ^  
make: *** [GNUmakefile:49: lwlibav_input.o] Error 1  
make: *** Waiting for unfinished jobs....  
```

</p>
</details>

--- a/AviUtl/lwlibav_input.c
+++ b/AviUtl/lwlibav_input.c
@@ -200,6 +200,7 @@ static void *open_file( char *file_path, reader_option_t *opt )
     /* Set options. */
     lwlibav_option_t lwlibav_opt;
     lwlibav_opt.file_path         = file_path;
+    lwlibav_opt.cache_dir         = NULL;
     lwlibav_opt.threads           = opt->threads;
     lwlibav_opt.av_sync           = opt->av_sync;
     lwlibav_opt.no_create_index   = opt->no_create_index;

